### PR TITLE
Added direct link to rsync mirror for gutenberg corpus generation

### PR DESCRIPTION
**Pull request by srisi**
Do not merge until approved by Keith and Mingfei.

`GUTENBERG_RSYNC_PATH` should point to the base of the rsync mirror

- Uses gutenberg_id to track down all useful filepaths

- Is there a reason why we shouldn't use properly encoded ASCII files? UTF-8 is just a superset, right? So it shouldn't produce any conflicts. I coded it so it would first look for UTF-8 but also accept ASCII files.

- More notes coming--but right now I need to get home.

### CODE REVIEW

**Comment by mscuthbert** [comment]
Ascii files should be fine.

File: gender_novels/corpus_gen.py [hunk 1]
```diff
@@ -47,22 +51,34 @@ def generate_corpus_gutenberg():
...
+    filepaths = []
+    for gutenberg_id in range(200):
+        novel_directory = generate_gutenberg_rsync_path(gutenberg_id)
+        utf8_path = novel_directory.joinpath(f'{gutenberg_id}-0.txt')
+        if os.path.isfile(utf8_path):
+            filepaths.append(utf8_path)
+            continue
+        ascii_path = novel_directory.joinpath(f'{gutenberg_id}.txt')
+        if os.path.isfile(ascii_path):
+            filepaths.append(ascii_path)
```
Comment (equationcrunchor) [inline gender_novels/corpus_gen.py]: Maybe also consider what to do if there is no book with a certain Gutenberg ID, although it seems that every ID corresponds to a book.

**Comment by equationcrunchor** [comment]
Looks pretty good.  I think I might change the get_novel_text_gutenberg function to only convert the file if there is an exception because detecting the encoding is one of the slowest steps, and also because right now it's set up to convert ASCII to UTF-8. That should work, right?

**Comment by srisi** [comment]
@equationcrunchor: That's a good idea. I'll implement that tomorrow morning.

**Comment by equationcrunchor** [comment]
So we are discarding novels without dates?

**Comment by mscuthbert** [comment]
I think we should.

**Pull request merged**